src/routes/reference/bufnmf.svx

```diff
@@ -9,4 +9,65 @@ tags:
     - spectral
 flair: reference
 category: Decompose
----
\ No newline at end of file
+---
+
+<script>
+    import IndentNote from '$lib/components/IndentNote.svelte';
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+</script>
+
+## BufNMF
```

Comment:
The layout will already render this for you so you can omit it or start with "introduction" or something else

Reply:
ah i understand.